ConsoleApp/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿//Task #1
+var x = 10;
+var y = 12;
+var sum = 0;
+if (x > y)
+{
+    for(int i = y;  i <= x; i++)
+    {
+        sum += i;
+    }
+    System.Console.WriteLine($"Sum of the elements from y to x equals = {sum}");
+}
+else if (x < y)
+{
+    for (int i = x; i <= y; i++)
+    {
+        sum += i;
+    }
+    System.Console.WriteLine($"Sum of the elements from x to y equals = {sum}");
+}
+else 
```

Comment:
you do not need this else branch because if x == y your for loop would have single iteration